coverage/main/main.go

```diff
@@ -1,10 +1,11 @@
+//go:build testrunmain
```

Comment:
This is out of scope (altho arguably also need to be done).

If you want to make an other PR which just `go fmt ./...` with go1.18 I'll review it.

Reply:
Someone else beat me to it. Rebased and this went away.